media-sound/fluidsynth/files/2.3.2-musl-implicit-declaration-fix.patch

```diff
@@ -0,0 +1,20 @@
+https://github.com/FluidSynth/fluidsynth/pull/1244
+From: Brahmajit Das <brahmajit.xyz@gmail.com>
+Date: Thu, 11 May 2023 20:56:50 +0530
+Subject: [PATCH] Fix implicit declaration of function strcasecmp
+
+Signed-off-by: Brahmajit Das <brahmajit.xyz@gmail.com>
+Bug: https://bugs.gentoo.org/906150
+--- a/src/utils/fluidsynth_priv.h
++++ b/src/utils/fluidsynth_priv.h
+@@ -43,6 +43,7 @@
+
+ #if HAVE_STRING_H
+ #include <string.h>
```

Comment:
Actually, is there a HAVE_STRINGS_H? It should go under that (might need to add it to configure)

Reply:
On it.

Reply:
How about now?